### PR TITLE
New File High Low

### PR DESCRIPTION
**Pull request by KewlStatics**
New Code

### CODE REVIEW

File: chat-plugins/highlow.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+        },
+
+        low: function(target, room ,user) {
+            if (room.id !== 'casino') return this.errorReply("Higher or Lower can only be played in the Casino.");
+            if (!this.canTalk()) return this.errorReply("/highlow low - Access denied.");
+            if (!room.highlow) return this.errorReply("There is no highlow game in this room.");
+            Db('money').set(user.userid, Db('money').get(user.userid) - 1);
+            
+            if (result = high) {
```
Comment (SilverTactic) [inline chat-plugins/highlow.js:74]: = assigns values to things, === compares values
=== is what you need here

File: chat-plugins/highlow.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                return this.sendReplyBox('You has been dealt a ' +  this.rounds[] + '. Too bad, you lost <b><font color="' + color(user) + '">' + user + '</font></b>. Better luck next time!</font>');
+                delete room.highlow;
+            }
+            if (result=low) {
```
Comment (SilverTactic) [inline chat-plugins/highlow.js:78]: same here

File: chat-plugins/highlow.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+            Db('money').set(user.userid, Db('money').get(user.userid) - 1);
+            
+            if (result = low) {
+                return this.sendReplyBox('You has been dealt a ' +  this.rounds[] + '. Too bad, you lost <b><font color="' + color(user) + '">' + user + '</font></b>. Better luck next time!</font>');
+                delete room.highlow;
+            }
+            if (result=high) {
+                if (this.rounds[5]) return this.sendReplyBox()
+                Db('money').set(user.userid, Db('money').get(user.userid, 0) + 11);
+                if (!this.rounds[5]) return this.send ReplyBox(display)
```
Comment (Torpidwaters) [inline chat-plugins/highlow.js:61]: you cannot have a space here

File: chat-plugins/highlow.js [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+            Db('money').set(user.userid, Db('money').get(user.userid) - 1);
+            
+            if (result = high) {
+                return this.sendReplyBox('You has been dealt a ' +  this.rounds[] + '. Too bad, you lost <b><font color="' + color(user) + '">' + user + '</font></b>. Better luck next time!</font>');
+                delete room.highlow;
+            }
+            if (result=low) {
+                if (this.rounds[5]) return this.sendReplyBox()
+                Db('money').set(user.userid, Db('money').get(user.userid, 0) + 11);
+                if (!this.rounds[5]) return this.send ReplyBox(display)
```
Comment (Torpidwaters) [inline chat-plugins/highlow.js:81]: you can't have one here either

**Comment by Torpidwaters** [comment]
you should really test your code, or at least run it though something to check for syntax errors and other basic mistakes before you make a pr

File: chat-plugins/highlow.js [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+        },
+
+        high: function(target, room ,user) {
+            if (room.id !== 'casino') return this.errorReply("Higher or Lower can only be played in the Casino.");
+            if (!this.canTalk()) return this.errorReply("/highlow high - Access denied.");
+            if (!room.highlow) return this.errorReply("There is no highlow game in this room.");
+            Db('money').set(user.userid, Db('money').get(user.userid) - 1);
+            
+            if (result = low) {
+                return this.sendReplyBox('You has been dealt a ' +  this.rounds[] + '. Too bad, you lost <b><font color="' + color(user) + '">' + user + '</font></b>. Better luck next time!</font>');
```
Comment (Torpidwaters) [inline chat-plugins/highlow.js:55]: specify an array index

File: chat-plugins/highlow.js [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+        },
+
+        low: function(target, room ,user) {
+            if (room.id !== 'casino') return this.errorReply("Higher or Lower can only be played in the Casino.");
+            if (!this.canTalk()) return this.errorReply("/highlow low - Access denied.");
+            if (!room.highlow) return this.errorReply("There is no highlow game in this room.");
+            Db('money').set(user.userid, Db('money').get(user.userid) - 1);
+            
+            if (result = high) {
+                return this.sendReplyBox('You has been dealt a ' +  this.rounds[] + '. Too bad, you lost <b><font color="' + color(user) + '">' + user + '</font></b>. Better luck next time!</font>');
```
Comment (Torpidwaters) [inline chat-plugins/highlow.js:75]: specify one here as well

File: chat-plugins/highlow.js [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+        return win;
+    }
+}
+
+exports.commands = {
+    
+        deal: function(target, room ,user) {
+            if (room.id !== 'casino') return this.errorReply("Higher or Lower can only be played in the Casino.");
+            if (!this.canTalk()) return this.errorReply("/highlow deal - Access denied.");
+            if (Db('money').get(user.userid, 0) < 1) return this.errorReply("You don't have enough bucks to join this game.");
```
Comment (Torpidwaters) [inline chat-plugins/highlow.js:40]: We use bucks in the db not money

File: chat-plugins/highlow.js [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+            room.highlow.started = true;
+            room.highlow.startTime = Date.now();
+            return this.sendReplyBox(startDisplay);
+        },
+
+        high: function(target, room ,user) {
+            if (room.id !== 'casino') return this.errorReply("Higher or Lower can only be played in the Casino.");
+            if (!this.canTalk()) return this.errorReply("/highlow high - Access denied.");
+            if (!room.highlow) return this.errorReply("There is no highlow game in this room.");
+            Db('money').set(user.userid, Db('money').get(user.userid) - 1);
```
Comment (Torpidwaters) [inline chat-plugins/highlow.js:52]: bucks here, and everywhere else too, i'm not gonna comment every one

File: chat-plugins/highlow.js (hunk 2 above)
Comment (panpawn) [reply chat-plugins/highlow.js:78]: spacing too

File: chat-plugins/highlow.js (hunk 3 above)
Comment (panpawn) [reply chat-plugins/highlow.js:61]: also missing semi-colon

File: chat-plugins/highlow.js (hunk 4 above)
Comment (panpawn) [reply chat-plugins/highlow.js:81]: semi missing here, too

**Comment by panpawn** [comment]
https://travis-ci.org/jd4564/Pokemon-Showdown/builds/158785780

**Comment by panpawn** [comment]
@KewlStatics - you seemed to have abandoned this PR.

If you're still planning on working on it, let me know and I'll reopen it. Until then, this is getting closed.

**Pull request closed without merging**